### PR TITLE
My first two cents

### PR DESCRIPTION
**Pull request by kiwirives**
I restrained myself to change every curly to cuddle...

### CODE REVIEW

**Comment by MASACR99** [comment]
Visual studio changes them automatically when I format so they are gonna stay that way :3

**Comment by kiwirives** [comment]
But did you just ignore all the changes I pulled??? Review!!!

**Comment by MASACR99** [comment]
I'm reviewing it and implementing it on my code to avoid merging your comments since I already updated the code from when you added the comments

File: CompareImage.cpp [hunk 1]
```diff
@@ -39,19 +39,18 @@ void search(const std::string& searc_path, const int options)
 	//Search for all the files and directories inside the firstly specified path
 	std::cout << "Looking for images in folders and loading them...\n";
 	for (const auto& el : recursive_directory_iterator(searc_path, directory_options::skip_permission_denied))
-	{
+	{ //FIXME do you REALLY want a line in your code that's only a curly brace? CURLY BRACES IN A NEW LINE ARE EVIL
 		if (is_directory(el) != true)
 		{
-			aux = el.path().extension().string();
-			if ((aux.compare(".jpg")) == 0 || (aux.compare(".jpeg")) == 0 || (aux.compare(".png")) == 0 || (aux.compare(".webp")) == 0  || (aux.compare(".tiff")) == 0 ||(aux.compare(".tif")) == 0)
+			aux = el.path().extension().string(); //FIXME trim first ".". if aux is a char* aux+1 could bypass "."
```
Comment (MASACR99) [inline CompareImage.cpp:45]: as you can see a few lines above aux is of typse std::string

File: CompareImage.cpp [hunk 2]
```diff
@@ -202,16 +205,18 @@ void hash_function(int start_point)
 					{
 						hasher = hasher + gray_image.getMat(cv::ACCESS_READ).at<uchar>(l, j); //will leave it like this for now, seems to work
 					}
-					hasher = hasher * hasher; //just to make hasher even more unique
+					hasher = hasher * hasher; //just to make hasher even more unique //FIXME does it really?
```
Comment (MASACR99) [inline CompareImage.cpp:208]: Yes, imagine there's 2 hashes that are very close together, if you multiply one for another they are gonna be further away from each other

**Pull request closed without merging**